### DISCUSSION TITLE
HA notification before/after noob question

### DISCUSSION THREAD

**Discussion by R5fan** [blakeblackshear/frigate#1222] [Ask A Question] [answered]
The documentation provides examples of how to get a notification with image that "updates" to a better one. While this is great, and I do get 1 visible notification per event on my phone,  Id rather not use it, as it means getting 2 or 3 audio notifications every time its updated. I cant figure out what to change to prevent this?

**Comment by blakeblackshear** [comment, marked as answer]
Set a condition on your automation to only send when you want. The options for type are new/end/update.

**Comment by R5fan** [comment]
Thanks for the super quick response, that did the trick! For anyone else wondering, just add this condition:
```
condition: template
value_template: '{{ trigger.payload_json[''type'']== ''new'' }}'
```
(replace "new" with "end" if you want a slower notification with the best image).
